### ISSUE TITLE
bb production for deploy master branch

### ISSUE THREAD

**Issue by abtris** [apiaryio/black-belt#98]
If some tests failed we need only deploy command with announcement in Slack after tests passed (manual retry). 

Similar to `bb stage`, for example `bb production`.

**Comment by honzajavorek** [comment]
Related to #67?

**Comment by abtris** [comment]
@honzajavorek Yes, it's.

**Comment by abtris** [comment]
In version 0.12

**Comment by Almad** [comment]
Have you even tested?

```
arael-2:apiary-merger almad$ bb version
0.12.0
arael-2:apiary-merger almad$ bb production
Traceback (most recent call last):
  File "/usr/local/bin/bb", line 5, in <module>
    main()
  File "/Library/Python/2.7/site-packages/blackbelt/tasks.py", line 32, in main
    cli()
  File "/Library/Python/2.7/site-packages/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/Library/Python/2.7/site-packages/click/core.py", line 991, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Python/2.7/site-packages/click/core.py", line 971, in invoke
    return Command.invoke(self, ctx)
  File "/Library/Python/2.7/site-packages/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Python/2.7/site-packages/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/blackbelt/commands/production.py", line 12, in cli
    deploy_production()
  File "/Library/Python/2.7/site-packages/blackbelt/deployment.py", line 15, in deploy_production
    post_message("@here Deploying to production" % branch_name, "#deploy-queue")
NameError: global name 'branch_name' is not defined
arael-2:apiary-merger almad$ 
```

**Comment by abtris** [comment]
Fixed in 0.12.1
